DEPLOYNOTES.rst

```diff
@@ -3,6 +3,17 @@
 Deploy and Upgrade notes
 ========================
 
+1.8
+---
+
+* The minimum required Solr version has been bumped to 9.2. Please upgrade to this
+  version, copy all files under `solr_conf/conf/` to the configured Solr configset,
+  and restart Solr to ensure the managed schema is loaded. Then update the schema
+  and reindex::
+
+    python manage.py solr_schema
```

Comment:
I don't think we use this `solr_schema` command anymore. For us this should be handled by the ansible deploy, but we'll need to make sure to run setup tasks - not sure if it makes sense to document that here.  This documentation is probably enough as a reminder about the change.
```suggestion
```